internal/models/makemetric.go

```diff
@@ -123,11 +122,9 @@ func makemetric(
 		case float64:
 			// NaNs are invalid values in influxdb, skip measurement
 			if math.IsNaN(val) || math.IsInf(val, 0) {
-				if debug {
```

Comment:
It was caught. I commented on the old code. Just making sure this was the intended behavior.